nvflare/app_common/statistics/numeric_stats.py

```diff
@@ -139,16 +135,16 @@ def get_min_or_max_values(
     return global_metrics
 
 
-def bins_to_dict(bins: List[Bin]) -> Dict[BinRange, float]:
+def bins_to_dict(bins: List[Bin], precision: int) -> Dict[BinRange, float]:
     buckets = {}
     for bucket in bins:
-        bucket_range = BinRange(bucket.low_value, bucket.high_value)
+        bucket_range = BinRange(round(bucket.low_value, precision), round(bucket.high_value, precision))
```

Comment:
I think we should not add "round" into "bins_to_dict" as reasoning in another thread